src/_components/RoutesComponent.jsx

```diff
@@ -1,59 +1,55 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Route, Switch } from 'react-router-dom';
 
 import AuthenticateComponent from './AuthenticateComponent';
-import AssetTypesComponent from '../components/AssetTypesComponent';
-import AssetsComponent from '../components/AssetsComponent';
+import AssetTypes from '../components/AssetTypesComponent';
+import Assets from '../components/AssetsComponent';
 import LoginComponent from '../components/LoginComponent';
 import DashboardComponent from '../components/DashboardComponent';
-import FeedbackComponent from '../components/FeedbackComponent';
+import Feedback from '../components/FeedbackComponent';
 import AddAssetContainer from './AddAsset/AddAssetContainer';
 
-class RoutesComponent extends React.Component {
-  checkAuthentication = () => {
-    return !!(localStorage.getItem('art-prod-web-token'));
-  }
-  render () {
+class RoutesComponent extends Component {
+  checkAuthentication = () => !!(localStorage.getItem('token'))
```

Comment:
The token is now called [art-prod-web-token](https://github.com/AndelaOSP/art-dashboard/blob/7eb6835bb59a45393b8adc3111d4f0c12809b107/src/components/LoginComponent.jsx#L19)